### PR TITLE
Amb 474

### PR DESCRIPTION
**Pull request by melnicenkovadik**

### CODE REVIEW

File: src/components/StakingChart/index.js [hunk 1]
```diff
@@ -1,46 +1,101 @@
...
-//   );
-// }
+import React from 'react';
+import { AreaChart, Area, XAxis, Tooltip } from 'recharts';
+
+const data = [
+  {
+    name: 'Dec 11',
+    uv: 1500,
+  },
```
Comment (valar999) [inline src/components/StakingChart/index.js:8]: Should "Dec 11" be hardcoded here?

File: src/pages/Staking/components/InfoBlock/InfoBlock.js [hunk 2]
```diff
@@ -1,19 +1,14 @@
+/*eslint-disable*/
```
Comment (valar999) [inline src/pages/Staking/components/InfoBlock/InfoBlock.js:1]: Do not disable linter

Comment (melnicenkovadik) [reply src/pages/Staking/components/InfoBlock/InfoBlock.js:1]: i fix it

File: src/components/StakingChart/index.js (hunk 1 above)
Comment (melnicenkovadik) [reply src/components/StakingChart/index.js:8]: I'm call Vitalii tomorrow and ask for this data

File: package.json [hunk 3]
```diff
@@ -22,10 +22,10 @@
   "dependencies": {
     "@inc4/eslint-config-react": "1.0.1",
     "@kunukn/react-collapse": "^2.2.9",
-    "@usedapp/core": "^0.7.2",
+    "@usedapp/core": "~0.10.0",
```
[the remark this answers is not in the document]
Comment (melnicenkovadik) [reply package.json:25]: its a working)

**Comment by valar999** [comment]
I can't review it. You can merge if Andrew is ok with this.

**Pull request merged**